src/vs/base/browser/ui/scrollbar/scrollableElementOptions.ts

```diff
@@ -55,6 +55,11 @@ export interface ScrollableElementCreationOptions {
 	 * Defaults to 5.
 	 */
 	fastScrollSensitivity?: number;
+	/**
+	 * FastScrolling key
+	 * Defaults to 'Alt'.
+	 */
+	fastScrollKey?: string | undefined;
```

Comment:
We shouldn't merge this until we figure out where we want to put these settings https://github.com/microsoft/vscode/issues/78431#issuecomment-518980315

Reply:
We're just going to have other components use the editor setting for now